boards/shields/nrf2220ek/nrf2220ek.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_BOOT_BANNER=n
```

Comment:
Are these options necessary for nRF52 series too?

Reply:
This configuration is needed unconditionally for nRF53 and in some cases for nRF52 too. After F2F discussion we decided to keep it as a limitation.